acados/sim/sim_common.h

```diff
@@ -45,43 +49,45 @@ typedef struct {
     double *S_forw;  // forward seed
     double *S_adj;   // backward seed
 
-    casadi_function_t vde;
-    void (*forward_vde_wrapper)(const int, const int, const double *, double *, casadi_function_t);
-
-    casadi_function_t vde_adj;
-    void (*adjoint_vde_wrapper)(const int, const int, const double *, double *, casadi_function_t);
-
-    casadi_function_t jac;
-    void (*jacobian_wrapper)(const int, const double *, double *, casadi_function_t);
-
-    casadi_function_t hess;
-    void (*Hess_fun)(const int, const int, const double *, double *, casadi_function_t);
-
-    casadi_function_t impl_ode;
-    void (*eval_impl_res)(const int, const int, const double *, double *, casadi_function_t); // function pointer to residuals of implicit ode
-
-    casadi_function_t impl_jac_x;
-    void (*eval_impl_jac_x)(const int, const int, const double *, double *, casadi_function_t); // function pointer to jacobian of implicit ode
-
-    casadi_function_t impl_jac_xdot;
-    void (*eval_impl_jac_xdot)(const int, const int, const double *, double *, casadi_function_t); // function pointer to jacobian of implicit ode
-
-    casadi_function_t impl_jac_u;
-    void (*eval_impl_jac_u)(const int, const int, const double *, double *, casadi_function_t); // function pointer to jacobian of implicit ode
+	/* external functions */
```

Comment:
Module specific function pointers.

Reply:
same comment applies, this is work-in-progress ;)